src/shared/lib/themes.ts

```diff
@@ -1,11 +1,14 @@
 import { Theme } from '../types/museeks';
-// IMPROVE ME: scan the directory for all json files instead
-import lightTheme from '../themes/light.json';
-import darkTheme from '../themes/dark.json';
-import darkLegacyTheme from '../themes/dark-legacy.json';
+import fs from 'fs';
 
-export const themes = [
-  lightTheme as Theme,
-  darkTheme as Theme,
-  darkLegacyTheme as Theme,
-];
+const themes : Array<Theme> = [];
+fs.readdirSync('./src/shared/themes').forEach(name => {
```

Comment:
I am not 100% sure it would when the app is packaged. 

Can you try two things?

- `yarn build && yarn museeks`
- `yarn build && yarn package:l` (if you use linux)
  - try to install whatever package you are interested in (from `build/`)
  - run the app and see if something wrong happens

Reply:
I Tried to package the app and i found there is problem with the build and  i found it in master branch too
![image](https://github.com/martpie/museeks/assets/51155424/3a1f8549-9481-408e-a705-6bc000c795cd)

At first i thought maybe the missing modules is listed in `devDependency` instead in `dependency`, so i tried to look into package.json and yep that is not the problem because it listed in `dependency`
The app told pino module is missing. I tried to remove pino from the code, another module is missing too
![image](https://github.com/martpie/museeks/assets/51155424/4b34a35c-7d4a-4205-94da-66465b8ffb68)

So, the problem is the builder and i don't know it's only in linux or other device.

Reply:
Because this issue have solved, then i will fix the rest